syntax = "proto3";
package proto;
option go_package = "../pb";

import "proto/common.proto";


service FrameworkService {

  rpc   Detect(ServiceInput) returns (ServiceOutputBool);
  rpc   IsFrameworkUsed(ServiceInput) returns (ServiceOutputBool);
  rpc   PercentOfFrameworkUsed(ServiceInput) returns (ServiceOutputFloat);
}

//    protoc -I .  --proto_path=./proto    --go_out=./pb   --go-grpc_out=./pb ./proto/*.proto